lnbits/settings.py

```diff
@@ -1,92 +1,298 @@
+                        if type(value) == str:
+                            value = value.replace("'", "")
+                            values += f"'{value}'"
+                        values += ","
+                    q = ", ".join(keys)
+                    v = values.rstrip(",")
+                    sql = f"INSERT INTO admin.settings ({q}) VALUES ({v})"
+                    await db.execute(sql)
+                    logger.warning(
+                        "initialized admin.settings from enviroment variables."
+                    )
+
+                    row = await db.fetchone("SELECT * FROM admin.settings")
+                    assert row, "Newly updated settings couldn't be retrieved"
+
+                admin = Settings(**row)
+
+                for key, value in admin.dict(exclude_none=True).items():
+                    if not key in read_only_variables:
+                        try:
+                            setattr(settings, key, value)
+                        except:
+                            logger.error(
+                                f"error overriding setting: {key}, value: {value}"
+                            )
+
+                logger.debug(f"Admin settings:")
+                for key, value in settings.dict(exclude_none=True).items():
+                    logger.debug(f"{key}: {value}")
+
+                http = "https" if settings.lnbits_force_https else "http"
+                user = settings.lnbits_admin_users[0]
+
+                admin_url = (
+                    f"{http}://{settings.host}:{settings.port}/wallet?usr={user}"
+                )
+                logger.warning(f"✔️ Access admin user account at: {admin_url}")
+
+                if (
+                    settings.lnbits_saas_callback
+                    and settings.lnbits_saas_secret
+                    and settings.lnbits_saas_instance_id
+                ):
+                    with httpx.Client() as client:
+                        headers = {
+                            "Content-Type": "application/json; charset=utf-8",
+                            "X-API-KEY": settings.lnbits_saas_secret,
+                        }
+                        payload = {
+                            "instance_id": settings.lnbits_saas_instance_id,
+                            "adminuser": user,
+                        }
+                        try:
+                            client.post(
+                                settings.lnbits_saas_callback,
+                                headers=headers,
+                                json=payload,
+                            )
+                            logger.warning("sent admin user to saas application")
+                        except:
+                            logger.error(
+                                f"error sending admin user to saas: {settings.lnbits_saas_callback}"
+                            )
+
+            except:
+                logger.error("admin.settings tables does not exist.")
```

Comment:
- misleading message, this is not the only exception that can be raised

Reply:
This `try, except` block doesn't make sense. Just remove it, achieves the same thing.

Reply:
done